simple-serialisation-library/src/main/java/gaffer/serialisation/simple/TreeSetSerialiser.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package gaffer.serialisation.simple;
+
+import com.google.common.base.Splitter;
+import gaffer.commonutil.CommonConstants;
+import gaffer.exception.SerialisationException;
+import gaffer.serialisation.Serialisation;
+import java.io.UnsupportedEncodingException;
+import java.util.Iterator;
+import java.util.TreeSet;
+
+/**
+ * A <code>TreeSetSerialiser</code> is a serialiser for {@link TreeSet}s with
+ * {@link String} values.
+ */
+public class TreeSetSerialiser implements Serialisation {
```

Comment:
Can this be renamed to StringTreeSetSerialiser or some other variation that references that this serialiser will only serialise tree sets of type string (Or any object that has an accurate toString() method representation)